src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -2131,7 +2131,7 @@
     <!-- CloneMethod may be null in error scenarios-->
     <Field Name="CloneMethod" Type="MethodSymbol?" />
     <!-- Members and expressions passed as arguments to the With expression. -->
-    <Field Name="Arguments" Type="ImmutableArray&lt;(Symbol? Member, BoundExpression Expression)&gt;" />
+    <Field Name="Arguments" Type="ImmutableArray&lt;BoundExpression&gt;" />
```

Comment:
>Arguments [](start = 17, length = 9)

nit: If we model this node after BoundObjectCreationExpression, I don't think this field should be called "Arguments".
"InitializerExpression(s)" would seem more appropriate.

``` 
 <Node Name="BoundObjectCreationExpression" Base="BoundExpression">
   ...
   <Field Name="InitializerExpressionOpt" Type="BoundObjectInitializerExpressionBase?"/>
```

This can be PROTOTYPE'ed and addressed later.

Reply:
Doesn't matter to me